Big documentation update

MattyTheHacker opened the pull request.

MattyTheHacker committed:
Fix invite url docs

MattyTheHacker committed:
move terminology to new file

MattyTheHacker committed:
start contributing guides

MattyTheHacker committed:
formatting

MattyTheHacker committed:
Merge branch 'main' into docs-updates

MattyTheHacker committed:
add some stuff

MattyTheHacker committed:
add

MattyTheHacker committed:
add links

MattyTheHacker committed:
add example

MattyTheHacker committed:
add missing cogs from list

MattyTheHacker committed:
Merge branch 'main' into docs-updates

MattyTheHacker committed:
add link

MattyTheHacker committed:
fix link

MattyTheHacker committed:
add backtick

MattyTheHacker committed:
add some stuff

MattyTheHacker committed:
Merge branch 'main' into docs-updates

MattyTheHacker committed:
Merge branch 'main' into docs-updates

CarrotManMatt committed:
Merge branch 'main' into docs-updates

# Conflicts:
#	README.md

MattyTheHacker committed:
Merge branch 'main' into docs-updates

MattyTheHacker committed:
Merge branch 'main' into docs-updates

MattyTheHacker committed:
Merge branch 'main' into docs-updates

MattyTheHacker committed:
Add some stuff

MattyTheHacker committed:
Merge branch 'main' into docs-updates

MattyTheHacker committed:
fix warnings

MattyTheHacker committed:
remove empty whitespace

MattyTheHacker committed:
Merge branch 'main' into docs-updates

MattyTheHacker committed:
more terms

MattyTheHacker committed:
Add example cog

MattyTheHacker committed:
env example

MattyTheHacker committed:
response button

MattyTheHacker committed:
Merge branch 'main' into docs-updates

MattyTheHacker committed:
Merge branch 'main' into docs-updates

MattyTheHacker committed:
blah blah

MattyTheHacker committed:
db stuff first run

MattyTheHacker committed:
add gdpr warning

MattyTheHacker committed:
Merge branch 'main' into docs-updates

MattyTheHacker committed:
Merge branch 'main' into docs-updates

MattyTheHacker committed:
Merge branch 'main' into docs-updates

MattyTheHacker committed:
Merge branch 'main' into docs-updates

MattyTheHacker committed:
Merge branch 'main' into docs-updates

MattyTheHacker committed:
Merge branch 'main' into docs-updates

CarrotManMatt commented on TERMINOLOGY.md:
```diff
@@ -0,0 +1,93 @@
+#### [Message Commands](https://discord.com/developers/docs/interactions/application-commands#message-commands)
+
+
+Message commands, also known as `Message-Context Commands`, are executed via right-clicking on a [Message](https://discord.com/developers/docs/resources/message), clicking "Apps", then selecting the command from the menu.
```
```suggestion
Message-context commands, are executed via right-clicking on a [Discord *Message*](https://discord.com/developers/docs/resources/message), clicking "Apps", then selecting the command from the menu. The interaction callback method is provided information about which message was clicked, along with which user clicked it.
```

CarrotManMatt reviewed (changes requested):
One overall formatting suggestion is to make every sentence start on a new line. With paragraphs always separated by only 2 new lines (i.e., one single blank new line in between).

MattyTheHacker committed:
Apply suggestions from code review

Co-authored-by: Matt Norton <matt@carrotmanmatt.com>
Signed-off-by: Matty Widdop <18513864+MattyTheHacker@users.noreply.github.com>

pre-commit-ci-lite[bot] committed:
[pre-commit.ci lite] apply automatic fixes

MattyTheHacker committed:
Merge branch 'main' into docs-updates

cssbhamdev committed:
Merge main into docs-updates

cssbhamdev committed:
Merge main into docs-updates

CarrotManMatt committed:
Fix minor docs errors

cssbhamdev committed:
Merge main into docs-updates

cssbhamdev committed:
Merge main into docs-updates

Thatsmusic99 committed:
Allow committee-elect to update actions (and appear in auto-complete) (#508)

Signed-off-by: Holly <25277367+Thatsmusic99@users.noreply.github.com>
Co-authored-by: pre-commit-ci-lite[bot] <117423508+pre-commit-ci-lite[bot]@users.noreply.github.com>
Co-authored-by: Matty Widdop <18513864+MattyTheHacker@users.noreply.github.com>

cssbhamdev committed:
Merge main into docs-updates

cssbhamdev committed:
Merge main into docs-updates

cssbhamdev committed:
Merge main into docs-updates

cssbhamdev committed:
Merge main into docs-updates

cssbhamdev committed:
Merge main into docs-updates

cssbhamdev committed:
Merge main into docs-updates

cssbhamdev committed:
Merge main into docs-updates

cssbhamdev committed:
Merge main into docs-updates

cssbhamdev committed:
Merge main into docs-updates

cssbhamdev committed:
Merge main into docs-updates

cssbhamdev committed:
Merge main into docs-updates

cssbhamdev committed:
Merge main into docs-updates

cssbhamdev committed:
Merge main into docs-updates

cssbhamdev committed:
Merge main into docs-updates

cssbhamdev committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

Copilot committed:
Update documentation guides and terminology

Agent-Logs-Url: https://github.com/CSSUoB/TeX-Bot-Py-V2/sessions/9b0f58a5-8242-47a5-9226-548cda2938f0

Co-authored-by: MattyTheHacker <18513864+MattyTheHacker@users.noreply.github.com>

Copilot committed:
Refine contributing examples

Agent-Logs-Url: https://github.com/CSSUoB/TeX-Bot-Py-V2/sessions/9b0f58a5-8242-47a5-9226-548cda2938f0

Co-authored-by: MattyTheHacker <18513864+MattyTheHacker@users.noreply.github.com>

Copilot committed:
Adjust contributing examples

Agent-Logs-Url: https://github.com/CSSUoB/TeX-Bot-Py-V2/sessions/9b0f58a5-8242-47a5-9226-548cda2938f0

Co-authored-by: MattyTheHacker <18513864+MattyTheHacker@users.noreply.github.com>

Copilot committed:
Clarify docs examples

Agent-Logs-Url: https://github.com/CSSUoB/TeX-Bot-Py-V2/sessions/9b0f58a5-8242-47a5-9226-548cda2938f0

Co-authored-by: MattyTheHacker <18513864+MattyTheHacker@users.noreply.github.com>

MattyTheHacker committed:
Merge branch 'main' into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates

automatic-pr-updater[bot] committed:
Merge main into docs-updates
